### PR TITLE
Add a comment in some profiles to allow screen sharing

### PR DESCRIPTION
**Pull request by albinou**

### CODE REVIEW

**Review by rusty-snake — COMMENTED** [review]
:+1:

**Comment by albinou** [comment]
Oh, wait, chromium actually requires `whitelist ${RUNUSER}/pipewire-0`, sorry.

File: etc/profile-a-l/chromium-common.profile [hunk 1]
```diff
@@ -34,6 +34,9 @@ include whitelist-var-common.inc
 # if your kernel allows unprivileged userns clone.
 #include chromium-common-hardened.inc
 
+# Uncomment or put in your chromium-common.local to allow screen sharing under wayland.
```
Comment (kmk3) [inline etc/profile-a-l/chromium-common.profile:37]:
```suggestion
# Uncomment or put in your chromium-common.local to allow screen sharing under
# wayland.
```

wrap

File: etc/profile-a-l/firefox.profile [hunk 2]
```diff
@@ -41,6 +41,11 @@ dbus-user.own org.mpris.MediaPlayer2.firefox.*
 #dbus-user.own org.mpris.MediaPlayer2.plasma-browser-integration
 #dbus-user.talk org.kde.JobViewServer
 #dbus-user.talk org.kde.kuiserver
+# Uncomment or put in your firefox.local to allow screen sharing under wayland.
+#whitelist ${RUNUSER}/pipewire-0
+#dbus-user.talk org.freedesktop.portal.*
+# Also uncomment or put in your firefox.local if screen sharing sharing still does not work with the above lines (might depends on the portal implementation)
```
Comment (kmk3) [inline etc/profile-a-l/firefox.profile:47]:
```suggestion
# Also uncomment or put in your firefox.local if screen sharing sharing still
# does not work with the above lines (might depend on the portal
# implementation)
```

fix typo; wrap

**Comment by netblue30** [comment]
merged, thanks!

**Pull request merged**